arangod/Cluster/ClusterFeature.cpp

```diff
@@ -645,13 +645,16 @@ void ClusterFeature::start() {
   AsyncAgencyCommManager::INSTANCE->setSkipScheduler(false);
   ServerState::instance()->setState(ServerState::STATE_SERVING);
 
+#ifdef USE_ENTERPRISE
   // If we are on a coordinator, we want to have a callback which is called
   // whenever a hotbackup restore is done:
   if (role == ServerState::ROLE_COORDINATOR) {
     auto hotBackupRestoreDone = [this](VPackSlice const& result) -> bool {
```

Comment:
I know its not related to that PR but does the ClusterFeature and the ClusterInfo outlive this lambda?

Reply:
The lambda is registered as an AgencyCallback and stored in the ClusterFeature's `_hotbackupRestoreCallback` instance member. I guess this means the lambda will not outlive the ClusterFeature, at least.